src/System.Private.ServiceModel/src/System/ServiceModel/Description/CustomAttributeProvider.cs

```diff
@@ -83,7 +84,10 @@ public object[] GetCustomAttributes(Type attributeType, bool inherit)
                 case AttributeProviderType.MemberInfo:
                     return this.MemberInfo.GetCustomAttributes(attributeType, inherit).ToArray();
                 case AttributeProviderType.ParameterInfo:
-                    return this.ParameterInfo.GetCustomAttributes(attributeType, inherit).ToArray();
+                    //GetCustomAttributes could return null instead of empty collection for unknown reason here, workaround the issue by explicitly checking the null
+                    IEnumerable<Attribute> customAttributes = null;
```

Comment:
We need to understand why it is returning null, because an empty array may not be the appropriate thing to do. For history, look at #315.  In that case, an empty array was the wrong thing to do.